### ISSUE TITLE
Rationale behind sorting filtered options

### ISSUE THREAD

**Issue by thomaspaillot** [shentao/vue-multiselect#1763]
https://github.com/shentao/vue-multiselect/blob/f424a49bebfb8368e53bb35428ffa30d11ae8967/src/multiselectMixin.js#L20-L24

I would like to understand the rationale behind the sort mechanism of the filtered options. As I understand it, it is sorted by the option label length, it seems to me that this is quite an opiniated choice that will not produce good result in every cases. Would you be willing to introduce a new prop to provide a custom sort function or to remove the sort?

**Comment by dorep** [comment]
We have noticed the "issue". Can you please provide us with the option to NOT sort by length?

**Comment by zigcBenx** [comment]
It would be incredibly helpful if someone could implement prop for altering sorting mechanism. :pray:

**Comment by dorep** [comment]
Happy to see some movement on this :) The best!

**Comment by 5qemeister** [comment]
Yeah this will be good idea. Also waiting for some movement on this! :)
